library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipView.kt

```diff
@@ -72,8 +72,15 @@ class TooltipView @JvmOverloads constructor(
             with(imageViewTopArrow) {
                 visibility = tooltipViewState.getTopArrowVisibility()
                 setTint(tooltipViewState.getBackgroundColor())
-                layoutMarginStart(tooltipViewState.arrowMargin, tooltipViewState.getArrowPercentage())
-                setImageDrawable(ContextCompat.getDrawable(context, tooltipViewState.getTopArrowResource()))
+                val drawable = requireNotNull(
+                    ContextCompat.getDrawable(
+                        context,
+                        tooltipViewState.getTopArrowResource()
+                    )
+                )
```

Comment:
Why did you need this? `AppCompatImageView#setImageDrawable` accepts nullable `Drawable`s.

Reply:
It's highly unlikely that it will return null. However, in the rare case that it does, the `intrinsicWidth` behaves unexpectedly. Therefore, we intentionally want to trigger a crash in order to catch and handle this situation.